low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -45,24 +47,18 @@ def args(self) -> str:
     @staticmethod
     def usage(prog: str):
         """Print usage instructions."""
-        sys.stderr.write(
-            f"{prog} [setup [-p <LDR_R2>]|post_install [-p <LDR_R2>]|init [-dp] [-r <vm>]|config [-f] [-r <vm>]\n"
-            "|test [self|sanity]|reset [hard -p <LDR_R21>|soft]|join_cluster [-n <nodes>]\n"
-            "|manifest_support_bundle [<id>] [<path>]|support_bundle [<id>] [<path>]]\n"
-            "setup options:\n"
-            "\t -p Product name\n"
-            "join_cluster options:\n"
-            "\t -n Node names\n"
-            "init options:\n"
-            "\t -dp Create configured datapath\n"
-            "\t -r  Role to be configured on the current node\n"
-            "config options:\n"
-            "\t -f  Force reinitialization. Do not prompt\n"
-            "\t -r  Role to be configured on the current node"
-            "post_install options:\n"
-            "\t -p Product to be configured\n"
-            "reset options:\n"
-            "\t -p Product to be configured\n")
+        sys.stderr.write(f"""{prog}
+            [ -h|--help ]
+            [ post_install [<global_config_url>] ]
+            [ init [<global_config_url>] ]
+            [ config [<global_config_url>] ]
+            [ test [sanity|alerts] ]
```

Comment:
Looks fine.
 @ujjwalpl args for python modules beyond post_install, init, config are accepted beyond $global_config_url, as their arguments are specific to those scripts and not global or sspl config, which confstor can make available from, like for test,reset, manifest, suppprt_bundle, etc.